src/app/pages/MapPage/detail/DetailDefinitionList.tsx

```diff
@@ -43,7 +37,7 @@ function renderDescription(description: string, link?: LocationDescriptor | null
     )
   }
 
-  if (description.length > 300) {
+  if (description && description.length > 300) {
```

Comment:
We can use optional chaining here.

```suggestion
  if (description?.length > 300) {
```

Reply:
Object is possibly `undefined` dus kan niet..